samples/IISSample/project.json

```diff
@@ -23,5 +23,11 @@
     "include": [
       "web.config"
     ]
+  },
+  "tools": {
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-preview2-final"
```

Comment:
do we want to pin the version here? I remember seeing 1.0.0-* for tools in other packages.